MediaBrowser.Controller/MediaEncoding/EncodingHelper.cs

```diff
@@ -2653,26 +2653,121 @@ public string GetAudioFilterParam(EncodingJobInfo state, EncodingOptions encodin
 
             var filters = new List<string>();
 
-            if (channels.HasValue
-                && channels.Value == 2
-                && state.AudioStream is not null
-                && state.AudioStream.Channels.HasValue
-                && state.AudioStream.Channels.Value == 6)
+            if (channels is 2 && state.AudioStream?.Channels is > 2)
             {
                 if (!encodingOptions.DownMixAudioBoost.Equals(1))
                 {
                     filters.Add("volume=" + encodingOptions.DownMixAudioBoost.ToString(CultureInfo.InvariantCulture));
                 }
 
-                switch (encodingOptions.DownMixStereoAlgorithm)
+                switch (state.AudioStream.Channels.Value)
                 {
-                    case DownMixStereoAlgorithms.Dave750:
-                        filters.Add("pan=stereo|c0=0.5*c2+0.707*c0+0.707*c4+0.5*c3|c1=0.5*c2+0.707*c1+0.707*c5+0.5*c3");
+                    case 3:
+                    {
+                        switch (encodingOptions.DownMixStereoAlgorithm)
+                        {
+                            case DownMixStereoAlgorithms.Rfc7845:
+                                filters.Add("pan=stereo|FL=0.414214*FC+0.585786*FL|FR =0.414214*FC+0.585786*FR");
```

Comment:
Don't know if it matters but there is extra whitespace in the filter

Reply:
Should not affect functionality but removed for consistency